### DISCUSSION TITLE
Dynamic configuration

### DISCUSSION THREAD

**Discussion by Kickbuttowaski** [tailwindlabs/tailwindcss#8936] [Help]
Is there any way to configure the theme values dynamically in tailwind react ? I am getting my theme values (background color, color) from server. I need to update these values in tailwind config and use. Not able to find anything in the documentation also

Example: I am getting primary color as #bada55 from server I need to update this value in tailwind config color object.

Any workaround would also be helpful.

Thanks

**Comment by wongjn** [comment]
Use CSS variables. Print the colors from the server as CSS variables in some inline CSS or in a `style` attribute and then refer to these in your Tailwind config.

For example:

From server:
```css
:root {
  --color-primary: 200 100 50;
}
```
`tailwind.config.js`:
```js
module.exports = {
  theme: {
    colors: {
      primary: 'rgb(var(--color-primary) / <alpha-value>)',
    },
  },
};
```
Printing the color this way means that we can take advantage of the alpha channel modifiers in Tailwind. See the [documentation about CSS variable colors](https://tailwindcss.com/docs/customizing-colors#using-css-variables).
